example/src/index.js

```diff
@@ -1,17 +1,18 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import { Provider } from 'react-redux';
-import { BrowserRouter as Router } from 'react-router-dom';
-import '@babel/polyfill';
+import React from "react";
+import ReactDOM from "react-dom";
+import { Provider } from "react-redux";
+import { BrowserRouter as Router } from "react-router-dom";
+import "@babel/polyfill";
 
-import store from './lib/store';
-import App from './App';
+import store from "./lib/store";
+import App from "./App";
+import "./styles/index.scss";
 
```

Comment:
@byhag I believe our config enforces single quotes, can you run `yarn lint` from within `example`?